src/main/java/pl/isa/biblioteka/menu/BooksBorrowMenu.java

```diff
@@ -0,0 +1,106 @@
+package pl.isa.biblioteka.menu;
+
+import pl.isa.biblioteka.user.LogUser;
+import pl.isa.biblioteka.user.Person;
+
+import java.util.Scanner;
+
+import static pl.isa.biblioteka.books.BorrowBooks.*;
+
+public class BooksBorrowMenu {
+    public static Scanner scanner = new Scanner(System.in);
+
+    public void mainLoop() {
+        System.out.println("Co chcesz zrobić");
+        Option option = null;
+        do {
+            printMenu();
+            try {
+                option = chooseOption();
+                executeOption(option);
+            } catch (IllegalArgumentException e) {
+                System.out.println(e.getMessage());
+            }
+        } while (option != Option.EXIT);
+    }
+
+    private Option chooseOption() {
+        int option = scanner.nextInt();
+        scanner.nextLine();
+        return Option.fromInt(option);
+    }
+
+    private void executeOption(Option option) {
+        LogUser logUser = new LogUser();
+        switch (option) {
+            case BORROW_BOOK -> {
+                if ((addBookToPerson())) {
```

Comment:
Jako że tu powinno dojść wczytywanie, to fajnie żeby wszystkie case'y były tylko wywołaniem funkcji, czyli żeby nie było potrzeby korzystania z nawiasów { }

Reply:
poprawiono. w menu znajdują się tylko wywołania